dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/test-applications/TokenMinValidity0s.war/src/oidc/client/tokenMinValidity0s/servlets/TokenMinValidity0sServlet.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package oidc.client.tokenMinValidity0s.servlets;
+
+import jakarta.annotation.security.DeclareRoles;
+import jakarta.security.enterprise.authentication.mechanism.http.OpenIdAuthenticationMechanismDefinition;
+import jakarta.security.enterprise.authentication.mechanism.http.openid.ClaimsDefinition;
+import jakarta.security.enterprise.authentication.mechanism.http.openid.LogoutDefinition;
+import jakarta.security.enterprise.authentication.mechanism.http.openid.PromptType;
+import jakarta.servlet.annotation.HttpConstraint;
+import jakarta.servlet.annotation.ServletSecurity;
+import jakarta.servlet.annotation.WebServlet;
+import oidc.client.base.servlets.BaseServlet;
+
+@WebServlet("/TokenMinValidity0sServlet")
+@OpenIdAuthenticationMechanismDefinition(providerURI = "${providerBean.providerSecureRoot}/oidc/endpoint/OP1",
+                                         clientId = "client_1",
+                                         clientSecret = "mySharedKeyNowHasToBeLongerStrongerAndMoreSecureAndForHS512EvenLongerToBeStronger",
+                                         claimsDefinition = @ClaimsDefinition(callerNameClaim = "sub", callerGroupsClaim = "groupIds"),
+                                         redirectURI = "${baseURL}/Callback",
+                                         jwksReadTimeoutExpression = "${openIdConfig.jwksReadTimeoutExpression}",
```

Comment:
Don't think we should need `jwksReadTimeoutExpression` to be configured for these.

Reply:
`jwksReadTimeoutExpression` was added to all test apps using the `OpenIdAuthenticationMechanismDefinition` annotation to reduce random timeout failures on the build machines. `${openIdConfig.jwksReadTimeoutExpression}` resolves to 1min instead of the 500ms default when no jwks read timeout is configured.

Reply:
Oh yeah, that old chestnut... Ok, that's fine.